t/apicast-policy-url-rewriting.t

```diff
@@ -0,0 +1,236 @@
+use lib 't';
+use TestAPIcastBlackbox 'no_plan';
+
+repeat_each(1);
```

Comment:
Any reason why we can't have default 2 runs ?

Reply:
No. My bad. Copied it from the wrong place.